### PR TITLE
992: New AO examples

### PR DESCRIPTION
**Pull request by Katherine-Osos**
## Changes proposed in this pull request:
- Eligibility Page: updated AO examples for each org type
- Help > Domain Requests page: Updated AO verbiage to match new messaging used elsewhere

Related to #[992](https://github.com/cisagov/getgov/issues/992)

### CODE REVIEW

**Comment by Katherine-Osos** [comment]
Here are the two links for reviewing these content updates: 

- [Eligibility page](https://federalist-877ab29f-16f6-4f12-961c-96cf064cf070.sites.pages.cloud.gov/preview/cisagov/getgov-home/ko/new-ao-examples/domains/eligibility/)
- [Help>Domain Requests](https://federalist-877ab29f-16f6-4f12-961c-96cf064cf070.sites.pages.cloud.gov/preview/cisagov/getgov-home/ko/new-ao-examples/help/domain-requests/)

File: pages/domains/domains_eligibility.md [hunk 1]
```diff
@@ -50,9 +50,9 @@ When you request a .gov domain, we'll ask for information about your authorizing
 - [School districts](#school-districts)
 
 ### Executive branch federal agencies
-Domain requests from executive branch agencies must be authorized by CIOs or agency heads.
+Domain requests must be authorized by the agency's CIO or the head of the agency.
```
Comment (michelle-rago) [inline pages/domains/domains_eligibility.md:53]: My only comment is that I'd like the fed exec section to follow the pattern of the other sections by restating the org type. 

"Domain requests from executive branch federal agencies must..."

**Review by michelle-rago — APPROVED** [review]
Looks great, @Katherine-Osos!!

File: pages/domains/domains_eligibility.md (hunk 1 above)
Comment (Katherine-Osos) [reply pages/domains/domains_eligibility.md:53]: good catch! will make that update.

**Pull request merged**